uint16 col
uint16 row
---
float32 temperature
